go/file_manager/pfsserver/main/main.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
I think it's better to put main package files inside
`foo/cmd/cli/`
See https://github.com/coreos/etcd/tree/master/cmd for folder structure reference.